anvil_schema/fss_index_data_dictionary/fss_index_data_dictionary.json

```diff
@@ -0,0 +1,888 @@
+{
+  "version": 1,
+  "tables": [
+    {
+      "name": "anvil_activity",
+      /// ???
+      "description": "",
```

Comment:
A table describing the activities performed on biosamples or files that resulted in the generation of additional files. The anvil_activity table is used to capture activity provenance that does not fit into the other activity tables described in the schema (such as file indexing, file checksums, etc.).